obs-packaging/Dockerfile

```diff
@@ -1,8 +1,10 @@
-ARG SUSE_VERSION=${SUSE_VERSION:-42.3}
+ARG SUSE_VERSION=${SUSE_VERSION:-15.1}
```

Comment:
this looks like a downgrade

cc @marcov

Reply:
That's fine, numbering jumped from 42.3 to 15.0, go figure :laughing: